### PR TITLE
Update requirements, installation script

### PR DESCRIPTION
**Pull request by dimkarakostas**

### CODE REVIEW

File: backend/requirements.txt [hunk 1]
```diff
@@ -1,17 +1,7 @@
...
 PyYAML==3.11
-argh==0.26.1
-argparse==1.2.1
 coverage==4.2
 coveralls==1.1
 django-nose==1.4.4
-funcsigs==0.4
 mock==1.3.0
-pathtools==0.1.2
-pep8==1.7.0
```
Comment (dionyziz) [inline backend/requirements.txt]: Don't we need this for the syntax checks?

File: install.sh [hunk 2]
```diff
@@ -2,10 +2,10 @@
 
 function install_nodejs_npm {
     sudo apt-get install curl
-    curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -   ## install or update nodejs with npm
+    curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -   ## install or update nodejs with npm
```
Comment (dionyziz) [inline install.sh]: What do you say about switching to nvm for this?

Comment (dimkarakostas) [inline install.sh]: Sure, let's open a new PR for this.

File: backend/requirements.txt (hunk 1 above)
Comment (dimkarakostas) [inline backend/requirements.txt]: Technically, no. pep8 is already installed explicitly on test time [here](https://github.com/dionyziz/rupture/blob/develop/.travis.yml#L4), so this requirement is covered as far as CI is concerned. Developers are required to follow the pep8 guidelines, but it is up to each one how they install and perform the checks while coding, it is not necessary for pep8 to be here in order to perform these checks.

Comment (dionyziz) [inline backend/requirements.txt]: This isn't satisfactory. `make all` should be able to run on a developer machine provided they have installed all dependencies. These dependencies should include pep8 and must be included in requirements.txt.

Perhaps we could introduce a `make dependencies` command to run pip install for backend and sniffer and npm install for client and realtime? But I think pep8 should be in the dependencies of both sniffer and backend.

**Comment by dionyziz** [comment]
LGTM

**Pull request merged**